### ISSUE TITLE
[Explanation] Training slice-wise and predicting subject-wise?

### ISSUE THREAD

**Issue by apjklab** [MIC-DKFZ/TractSeg#240]
Hello, 

Firstly, thank you for your work and making it open source.

I had a doubt about the training process and combining the results as discussed in the TractSeg paper. 

Is the model trained slice-wise (so there is no need to combine and average the final 3D cube during training), basically a slice-wise loss is calculated with the groundtruth, but during inference, we predict for all slices, and then take mean across all axes. (also, this merging could then again be trained as a second FCNN)

Basically, the training is slice-wise, but during test time the prediction is subject-wise (by that, I mean, during prediction, all slices of a subject, for all axes are input, and the predictions obtained are combined using averaging.)

Is this understanding correct?

Thanks,
Joshi

**Comment by apjklab** [comment]
Thanks a lot for your help!

**Comment by mrrestre** [comment]
Hello! I have another question to the training process. When training with new data as explained in the Readme one is only performing the training on the first FCNN letting the second one really only as an additional option or is the training of the merging directly implemented?
